src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -354,21 +377,15 @@ export default class WebGLRenderer extends SystemRenderer
         if (renderTexture)
         {
             const baseTexture = renderTexture.baseTexture;
-            const gl = this.gl;
 
             if (!baseTexture._glRenderTargets[this.CONTEXT_UID])
             {
-                this.textureManager.updateTexture(baseTexture);
-                gl.bindTexture(gl.TEXTURE_2D, null);
-            }
-            else
-            {
-                // the texture needs to be unbound if its being rendererd too..
-                this._activeTextureLocation = baseTexture._id;
-                gl.activeTexture(gl.TEXTURE0 + baseTexture._id);
-                gl.bindTexture(gl.TEXTURE_2D, null);
+                // bind the current texture
+                this.textureManager.updateTexture(baseTexture, 0);
             }
 
+            this.unbindTexture(baseTexture);
```

Comment:
it does! we need to make sure the render texture itself is not bound as at this point we are rendering to it. WebGL throws lots of errors if you are rendering to a frame buffer whilst also having its texture bound. 
unbindTextutre makes sure that it happens.